packages/expo-updates/e2e/fixtures/Updates.e2e.ts

```diff
@@ -372,10 +372,6 @@ describe('Basic tests', () => {
   });
 
   it('supports rollbacks', async () => {
-    if (platform === 'ios') {
```

Comment:
You also need to change line 11 to set protocol version 1 for iOS

Reply:
Technically it was serving the rollbacks and the test was passing since rollbacks aren't manifests (and server.ts only uses protocol version for manifests). But nice catch, updating now.